debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -928,25 +928,28 @@ public static final Field MASK_COLUMN(int length) {
                                                                 DatabaseHistory.DDL_FILTER);
 
     private final SnapshotLockingMode snapshotLockingMode;
-    @Deprecated
-    private final boolean snapshotMinimalLocking;
 
     public MySqlConnectorConfig(Configuration config) {
         super(config);
 
-        this.snapshotLockingMode = SnapshotLockingMode.parse(config.getString(SNAPSHOT_LOCKING_MODE), SNAPSHOT_LOCKING_MODE.defaultValueAsString());
-        this.snapshotMinimalLocking = config.getBoolean(MySqlConnectorConfig.SNAPSHOT_MINIMAL_LOCKING);
+        // If deprectated snapshot.minimal.locking property is explicitly configured
```

Comment:
@gunnarmorling This seemed like this was the best spot to coerce the old configuration value into the new one.  I was hoping I could do it in the MySQLConnectorConfig#validateSnapshotLockingMode() method, but the Configuration instance is essentially read only at that point.